src/cli/gentest/cli.py

```diff
@@ -9,16 +9,12 @@
 from typing import TextIO
 
 import click
-import jinja2
 
 from ethereum_test_base_types import Hash
 
-from .request_manager import RPCRequest
+from .source_code_generator import get_test_source
 from .test_providers import BlockchainTestProvider
 
-template_loader = jinja2.PackageLoader("cli.gentest")
-template_env = jinja2.Environment(loader=template_loader, keep_trailing_newline=True)
-
 
 @click.command()
```

Comment:
I'm wondering how you'd like to design the interface once you implement different types of `blockchain_test`, for example, from multiple txs. Would you infer the type from the arguments (allow a list of tx, for example)? Or add sub-commands to gentext, like `gentest fromtx 0x....`?

Again, no need to answer in any depth here. Or implement it in the scope of this PR. Just thoughts for the process ))

Reply:
I have couple of ideas there. I will document it in the PR for #865.